package-lock.json

```diff
@@ -0,0 +1,7210 @@
+{
```

Comment:
Can you please delete the `package-lock.json`? Thanks!

Reply:
This is also an indicator that `npm` was used instead of `lerna`, which would have used `yarn` under the hood.

@alexkemboi4127 please check your tooling and use the right tools for the job!